backend/migrations/20241205034016-add-epversion-column.cjs

```diff
@@ -0,0 +1,31 @@
+require('dotenv').config();
+const { DataTypes, TEXT } = require('sequelize');
+
+async function up({ context: queryInterface }) {
+    await queryInterface.addColumn(
+        {schema: process.env.DATABASE_SCHEMA_NAME, tableName: 'arka_whitelist'},
```

Comment:
Pls remove this additional column and let's retain a single endpoint for whitelist which is common for both EP6 and EP7.